pkg/apis/v1beta1/ec2nodeclass_validation.go

```diff
@@ -128,6 +134,14 @@ func (in *SecurityGroupSelectorTerm) validate() (errs *apis.FieldError) {
 }
 
 func (in *EC2NodeClassSpec) validateAMISelectorTerms() (errs *apis.FieldError) {
+	if len(in.AMISelectorTerms) == 0 {
+		errs = errs.Also(apis.ErrMissingOneOf())
```

Comment:
Does `apis.ErrMissingOneOf()` receive an argument here? Or does it just work out of box since we are already inside of the amiSelectorTerms?

Reply:
Should have it out of the box since the path is injected at the call site for validateAMISelectorTerms.